### DISCUSSION TITLE
<!--[if BLOCK]><![endif]--> <!--[if ENDBLOCK]><![endif]--> inside textarea

### DISCUSSION THREAD

**Discussion by mtctonyhkhk2010** [livewire/livewire#7196] [7. Bugs (archived)]
### Livewire version

3.0.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. create a livewire component that contain a textarea and add @if @endif tag inside the the value area.
```
 <textarea>
@if($something_true)
@endif
</textarea>
```
2. `<!--[if BLOCK]><![endif]--> <!--[if ENDBLOCK]><![endif]-->` will be rendered inside of the textarea, like this
![image](https://github.com/livewire/livewire/assets/6291773/250a5190-4387-44de-8f67-c39edbc0298f)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Eaorns** [comment]
You can disable these markers by adding/changing `'inject_morph_markers' => false` in `config/livewire.php`.
[See documentation](https://livewire.laravel.com/docs/morphing#injecting-morph-markers)

**Reply by mtctonyhkhk2010** [reply]
But doing so will make livewire fail to properly detect conditionals like the doc said right? Is there other way to work around this?

**Reply by tikey23** [reply]
We have the same problem. Small workaround for some cases: {{ @$var }} when you just check for existing vars in your if-statement. 

Hope there will be a better solution.

**Comment by uandco** [comment]
Only the `@if` directive triggers `inject_morph_markers`. To avoid them, you can use a ternary operator inside a raw variable output:

```
<textarea>
{!! $something_true ? 'display something' : '' !!}
</textarea>
```
